apps/glusterfs/app_volume.go

```diff
@@ -252,17 +252,22 @@ func (a *App) VolumeDelete(w http.ResponseWriter, r *http.Request) {
 			return nil
 		}
 
-		if volume.Info.BlockInfo.BlockVolumes == nil {
```

Comment:
Does the range below work as expected if the BlockVolumes value is nil? (I honestly don't remember)

Reply:
yes, I checked the following in the sandbox in golang.org ;-)
```
package main

import "fmt"

func main() {
	fmt.Println("Hello!")
	
	var mylist []string
	mylist = nil
	
	for _, myitem := range mylist {
		fmt.Println("myitem: %v", myitem)
	}
	
	fmt.Println("Bye...")
}
```

Reply:
it prints
```
Hello!
Bye...
```

Reply:
Great!